packages/configtest/src/index.ts

```diff
@@ -13,22 +15,29 @@ class ConfigTestCommand {
                 pkg: '@webpack-cli/configtest',
             },
             [],
-            async (configPath: string) => {
+            async (configPath: string): Promise<void> => {
                 const { options } = await cli.resolveConfig({ config: [configPath] });
 
-                const isValidationError = (error) => {
+                const isValidationError = (error): boolean => {
                     // https://github.com/webpack/webpack/blob/master/lib/index.js#L267
                     // https://github.com/webpack/webpack/blob/v4.44.2/lib/webpack.js#L90
-                    const ValidationError: any = webpack.ValidationError || webpack.WebpackOptionsValidationError;
+                    //eslint-disable-next-line @typescript-eslint/no-explicit-any
+                    const webpackValidationError: any = ValidationError || WebpackOptionsValidationError;
 
-                    return error instanceof ValidationError;
+                    return error instanceof webpackValidationError;
                 };
 
                 //eslint-disable-next-line @typescript-eslint/no-explicit-any
-                const error: any = webpack.validate(options);
-
-                if (error) {
-                    logger.error(isValidationError(error) ? error.message : error);
+                const error: any = validate(options);
+
+                if (error && error.length) {
+                    if (isValidationError(error)) {
+                        // eslint-disable-next-line @typescript-eslint/ban-ts-ignore
+                        // @ts-ignore
+                        logger.error(isWebpack5 ? new ValidationError(error) : new WebpackOptionsValidationError(error));
```

Comment:
Why?

Reply:
else it logs - 

![Screenshot at 2021-01-05 19-26-58](https://user-images.githubusercontent.com/46647141/103654544-07371c00-4f8c-11eb-94b1-a3c5a796c3ee.png)

`migrate` uses the same logic https://github.com/webpack/webpack-cli/blob/19a25cf83dc2f680a5028f4b449d7f79895231f0/packages/migrate/src/index.ts#L133

Reply:
weird, I will look at this in near future